_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -59,54 +63,54 @@ By setting the `tabindex` [attribute value][] of an `iframe` element to `-1` or
 
 #### Passed Example 1
 
-This `iframe` element contains no content that is part of [sequential focus navigation][].
+This `iframe` element does not have a `tabindex` [attribute value][] that is a negative number
 
 ```html
-<iframe tabindex="-1" srcdoc="<h1>Hello world</h1>"></iframe>
+<iframe tabindex="0" srcdoc="<a href='/'>Home</a>"></iframe>
 ```
 
-#### Passed Example 2
+### Failed
 
-This `iframe` element contains a link that is not part of [sequential focus navigation][] because of its `tabindex`.
+#### Failed Example 1
+
+This `iframe` element contains a [visible][] link that is part of its [flattened tabindex-ordered focus navigation scope][]. Note that the link is not part of the document's [sequential focus navigation][] because the containing `iframe` is excluded from the document's [tabindex-ordered focus navigation scope](https://html.spec.whatwg.org/multipage/interaction.html#tabindex-ordered-focus-navigation-scope) due to its negative `tabindex`.
```

Comment:
I'm also considering whether that inlined note should be made into a full Background note (slightly changed), since it explain the full intend of the rule: stiff that is in the FTOFNS of an iframe should be in the tab-order of the web page.

Reply:
+1 to adding this as a background note